c10/core/CPUAllocator.h

```diff
@@ -6,6 +6,7 @@
 #include <c10/core/Allocator.h>
 #include <c10/util/Logging.h>
 #include <c10/util/numa.h>
+#include <c10/util/Unicode.h>
```

Comment:
To reduce unneeded header dependencies, please include Unicode.h only in .cpp file that uses it